madgraph/iolibs/export_v4.py

```diff
@@ -7616,6 +7627,7 @@ def create_couplings_main(self, nb_def_by_file=25):
         #                     double precision mu_r2, as2
         #                     include \'model_functions.inc\'""")
         # fsock.writelines("""include \'input.inc\'
+        #                     include \'../vector.inc\'
         #                     include \'coupl.inc\'
```

Comment:
same (again one which is less clear if we need it or not)

Reply:
same as above, vector.inc is needed by coupl.inc